addons/account/wizard/accrued_orders.py

```diff
@@ -81,22 +82,8 @@ def _compute_journal_id(self):
     @api.depends('date', 'journal_id', 'account_id', 'amount')
     def _compute_preview_data(self):
         for record in self:
-            preview_vals = [self.env['account.move']._move_dict_to_preview_vals(
-                record._compute_move_vals()[0],
-                record.company_id.currency_id,
-            )]
-            preview_columns = [
-                {'field': 'account_id', 'label': _('Account')},
-                {'field': 'name', 'label': _('Label')},
-                {'field': 'debit', 'label': _('Debit'), 'class': 'text-end text-nowrap'},
-                {'field': 'credit', 'label': _('Credit'), 'class': 'text-end text-nowrap'},
-            ]
-            record.preview_data = json.dumps({
-                'groups_vals': preview_vals,
-                'options': {
-                    'columns': preview_columns,
-                },
-            })
+            move_vals = record._get_move_vals()
+            record.preview_data = self._get_preview_data(move_vals, record.company_id.currency_id)
```

Comment:
```suggestion
            record.preview_data = self._get_preview_data(record._get_move_vals(), record.company_id.currency_id)
```
I don't think we need the variable. The function name is clear enough.